.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: node_js
 node_js:
     - "stable"
-    - "6.3.0"
+    - "8.6.0"
```

Comment:
TravisCI builds and tests the package in all the versions of NodeJS specified under node_js, so, stable is already 8.6.0.